### PR TITLE
Setup for multi frame subjects

### LINKED ISSUE #1344: Configuring the display of a multiFrame subject viewer
In the classification interface, a multiFrame subject viewer will display when:
- [ ] the workflow configuration is `{ subject_viewer: 'multiFrame' }`
or
- [ ]  if (1) the subject delivered to the client has multiple frames and (2) the workflow is not configured to display all frames at once

### PR DESCRIPTION
**Pull request by simensta**
_Please request review from `@zooniverse/frontend` team. If PR is related to design, please request review from `@beckyrother` in addition._ 

**Package:** lib-classifier

Closes #1344 

**Describe your changes:**
This PR sets up the folders, files, and logic for rendering the multiFrame subject viewer. For this PR, the MultiFrameViewer component itself is a simple placeholder.

The multiFrame viewer will render if:
-  there is a workflow configuration of `{ subject_viewer: 'multiFrame' }` or the subject has between 2 and 10 frames
- the workflow is not configured to display all frames at once

# Review Checklist

## General

- [x] Are the tests passing locally and on Travis?
- [x] Is the documentation up to date?

## Components
- [x] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [x] Is it passing accessibility checks in the storybook?

## Apps

- [x] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [x] Does it work on mobile?
- [x] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

File: packages/lib-classifier/src/store/Subject.spec.js [hunk 1]
```diff
@@ -57,6 +60,66 @@ describe('Model > Subject', function () {
...
+    it('should return a null viewer when workflow.configuration["enable_switching_flipbook_and_separate"] === "true"', function () {
+      const multiFrameSubject = SubjectFactory.build({ locations: [{ 'image/png': 'https://foo.bar/example.png' }, { 'image/png': 'https://foo.bar/example.png' }]})
+      const subjectStore = Subject.create(multiFrameSubject)
+      subjectStore.workflows = WorkflowStore.create({})
+      subjectStore.workflows.setResource(workflowWithConfigEnableSwitching)
+      subjectStore.workflows.setActive(workflowWithConfigEnableSwitching.id)
+      expect(subjectStore.viewer).to.be.null()
+    })
+
+    it('should return the multiFrame viewer if the workflow configuration for subject_viewer is defined as multiFrame', function () {
```
Comment (mcbouslog) [inline packages/lib-classifier/src/store/Subject.spec.js]: This is to test that regardless of the subject, including type or number of locations, if the workflow config is set to use multi frame viewer then the subject uses the multi frame viewer? Nice!

Comment (simensta) [reply packages/lib-classifier/src/store/Subject.spec.js]: It is, however, I am realizing there is an assumption that `{ subject_viewer: 'multiFrame' }` can only be enabled for subjects with between 2 and 10 frames.

Comment (simensta) [reply packages/lib-classifier/src/store/Subject.spec.js]: I will need to revisit this logic when creating a workflow config option in the lab editor. Thank you for the question!

Comment (srallen) [reply packages/lib-classifier/src/store/Subject.spec.js]: @beckyrother is working on some preliminary ideas for the workflow lab editor page with these new configurations.

**Pull request merged**